include/libopencm3/stm32/common/usart_common_fl_0.h

```diff
@@ -0,0 +1,286 @@
+/** @defgroup usart_defines USART Defines
+ *
+ * @brief <b>Defined Constants and Types Common between the STM32F0/STM32L0 USART</b>
+ *
+ * @ingroup STM32F0xx_defines
+ * @ingroup STM32L0xx_defines
+ *
+ * @version 1.1.0
+ *
+ * @date 1 Jan 2015
+ *
+ * LGPL License Terms @ref lgpl_license
+ */
+
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * This library is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @cond */
+#ifdef LIBOPENCM3_USART_H
+/** @endcond */
+
+#ifndef LIBOPENCM3_USART_FL_0_H
+#define LIBOPENCM3_USART_FL_0_H
+
+/*****************************************************************************/
+/* Register definitions                                                      */
+/* Common to both L0 and F0 variants                                         */
+/*****************************************************************************/
+
+#define USART_CR1(usart_base)		MMIO32(usart_base + 0x00)
+#define USART_CR2(usart_base)		MMIO32(usart_base + 0x04)
+#define USART_CR3(usart_base)		MMIO32(usart_base + 0x08)
+#define USART_BRR(usart_base)		MMIO32(usart_base + 0x0c)
+#define USART_GTPR(usart_base)		MMIO32(usart_base + 0x10)
+#define USART_RTOR(usart_base)		MMIO32(usart_base + 0x14)
+#define USART_RQR(usart_base)		MMIO32(usart_base + 0x18)
+#define USART_ISR(usart_base)		MMIO32(usart_base + 0x1c)
+#define USART_ICR(usart_base)		MMIO32(usart_base + 0x20)
+#define USART_RDR(usart_base)		MMIO16(usart_base + 0x24)
+#define USART_TDR(usart_base)		MMIO16(usart_base + 0x28)
```

Comment:
is 16 actually right? I know 8 was wrong, but is 16 actually correct?  This doesn't appear to be like the spi peripherals or dma where the size of the operation matters.